package.json

```diff
@@ -27,7 +30,7 @@
     "connect-redis": "^4.0.3",
     "cookie-parser": "~1.4.3",
     "csurf": "^1.10.0",
-    "csv-parse": "^1.2.0",
+    "csv-parse": "^4.6.3",
```

Comment:
Why can't we update the dependencies in a different PR? It seems a bit rushed if this is not necessary here...

Reply:
there were incompatibilities between the old versions itself that have been removed. however gulp-imagemin required a downgrade.

Reply:
Okay... why? I can live with the handlebars updates. Is csv-parse used somewhere? I thought all CSV parsing had moved to the server...

Reply:
that research was not part of this pr, but having all things working and no misconfiguration in versions only...